vault/request_forwarding_rpc.go

```diff
@@ -82,9 +81,9 @@ func (s *forwardedRequestRPCServer) Echo(ctx context.Context, in *EchoRequest) (
 		ReplicationState: uint32(s.core.ReplicationState()),
 	}
 
-	if raftStorage, ok := s.core.underlyingPhysical.(*raft.RaftBackend); ok {
-		reply.RaftAppliedIndex = raftStorage.AppliedIndex()
-		reply.RaftNodeID = raftStorage.NodeID()
+	if raftBackend := s.core.getRaftBackend(); raftBackend != nil {
```

Comment:
We probably only want to add these values to the reply if `!s.core.isRaftHAOnly()`

Reply:
Is there a reason why we don't want to include these values if we're HA only? Is this value only used for the two phased commit (which is not used if we're HA-only) or are there other more general uses for this?

Reply:
It's only used for the two phased commit

Reply:
Done in fd4ed94